From afd6e1e3bd01610839d96e86d685461842db89b4 Mon Sep 17 00:00:00 2001
From: Thomas Kemmer <thomas@binaryden.de>
Date: Thu, 30 Nov 2023 13:07:49 +0100
Subject: [PATCH] Disable Wayland support

---
 panels/display/cc-display-panel.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/panels/display/cc-display-panel.c b/panels/display/cc-display-panel.c
index 5a513b4..0c28b10 100644
--- a/panels/display/cc-display-panel.c
+++ b/panels/display/cc-display-panel.c
@@ -28,7 +28,6 @@
 
 #include <libcinnamon-desktop/cdesktop-enums.h>
 #include <math.h>
-#include <gdk/gdkwayland.h>
 
 #include <libupower-glib/upower.h>
 
@@ -114,7 +113,7 @@ struct _CcDisplayPanel
 
 CC_PANEL_REGISTER (CcDisplayPanel, cc_display_panel)
 
-#define WAYLAND_SESSION() (GDK_IS_WAYLAND_DISPLAY (gdk_display_get_default()))
+#define WAYLAND_SESSION() (0)
 
 static void update_bottom_buttons (CcDisplayPanel *panel);
 static void apply_current_configuration (CcDisplayPanel *self);
-- 
2.41.0

